source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,485 @@
+  schema enforcement, which the SDKs use to automatically sync objects 
+  across devices and handle conflict resolution.
+
+Regardless of which type of service you choose, Edge Servers use Device Sync 
+to sync data between the Edge Server instance and Atlas. Even when you choose 
+an Edge Server Service, you must first enable Device Sync before you can create 
+Edge Server instances.
+
+.. important:: You can't add Atlas Device SDK Apps later to an Edge Server Service
+
+   You can always add Edge Servers to a Device Sync Service. But you can't use
+   Atlas Device SDK connections with an Edge Server Service. 
+   
+   Users are not portable between services. If you start with an Edge Server 
+   Service, but later want to add an Atlas Device SDK application, there is no 
+   way to transfer any users in your Edge Service to a new Device Sync Service.
+
+You can create an Edge Server-compatible service in Data Services or App 
+Services. 
+
+- You might select Data Services when you just want to get started building
+  with Edge Server or Device Sync and Edge Server, and don't need other 
+  App Services.
+- You might select App Services when you want to bundle your service with 
+  authentication, Functions, Triggers, HTTPS Endpoints, or Values and Secrets.
+
+.. tabs::
+
+   .. tab:: Create a Data Service
+      :tabid: atlas-service
+
+      .. include:: /includes/create-service-data-services.rst
+
+      .. include:: /includes/edge-server-enable-device-sync.rst
+
+   .. tab:: Create an App Service
+      :tabid: app-service
+
+      Refer to the documentation to :ref:`create-app`.
+
+      When you choose your App type, select ``Edge Server`` or ``Device Sync``. 
+
+      .. include:: /includes/edge-server-enable-device-sync.rst
+
+Supported Authentication Providers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Edge Server supports a subset of authentication providers:
+
+- :ref:`anonymous-authentication`
+- :ref:`email-password-authentication`
+- :ref:`custom-jwt-authentication`
+- :ref:`api-key-authentication`
+
+You must enable at least one of these providers to enable clients to connect
+to the Edge Server.
+
+.. _create-edge-server-instance:
+
+Create an Edge Server Instance
+------------------------------
+
+After creating an Edge Server Service, you can add Edge Servers to it. The
+service can handle multiple Edge Servers, each of which has its own 
+registration key.
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      .. procedure::
+
+         .. step:: Add a server
+
+            Click the :guilabel:`Add a server` button to walk through the 
+            process of configuring your new Edge Server instance.
+
+            If this is the first server in your service, you're prompted to 
+            select a cluster, connect a data source, and optionally specify 
+            which collections to sync to the Edge Server. This configuration
+            applies to all Edge Servers in this service.
+
```

Comment:
The cluster used is only when setting up the Edge Server service. When going to add an Edge Server (whether the first or not) while already in the service/application page doesn't require setting this up. It will use the same cluster that device sync is using.